packages/dcos-integration-test/extra/test_dcos_diagnostics.py

```diff
@@ -635,9 +629,9 @@ def validate_units(units):
 
         # a unit must have all 3 fields not empty
         assert unit['id'], 'id field cannot be empty'
-        assert unit['name'], 'name field cannot be empty'
+        assert unit['name'], 'name field cannot be empty in {}'.format(unit)
```

Comment:
Most tests fail because the name is `''`. @janisz, how is this name determined?

Reply:
It's set to empty here https://github.com/dcos/dcos-diagnostics/blob/22809babf16bd5bd310a741c861c0acef0c31dd3/api/health_windows.go#L54

Reply:
I'm not really sure I understand. It seems `prettyName` is empty because `s := strings.Split(propsResponse.Description, ": ")` is empty. But how is `propsResponse` determined?

Reply:
It worked.